src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -520,29 +520,54 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
     if (useLpt) {
         CPU_LPT_SCOPE(LowPrecisionTransformations_Part4);
         OV_ITT_SCOPE(FIRST_INFERENCE, itt::domains::intel_cpu_LT, "LowPrecisionTransformations");
-
-        auto supportedPrecisions = std::vector<PrecisionsRestriction>({
-            PrecisionsRestriction::create<ngraph::opset1::Convolution>({
-                {0, {ngraph::element::u8, ngraph::element::i8}},
-                {1, {ngraph::element::i8}},
-            }),
-            PrecisionsRestriction::create<ngraph::opset1::ConvolutionBackpropData>({
-                {0, {ngraph::element::u8, ngraph::element::i8}},
-                {1, {ngraph::element::i8}}
-            }),
-            PrecisionsRestriction::create<ngraph::opset1::GroupConvolution>({
-                {0, {ngraph::element::u8}},
-                {1, {ngraph::element::i8}}
-            }),
-            PrecisionsRestriction::create<ngraph::opset1::Multiply>({
-                {0, {ngraph::element::u8}},
-                {1, {ngraph::element::i8}},
-            }),
-            PrecisionsRestriction::create<ngraph::opset1::MatMul>({
-                {0, {ngraph::element::u8, ngraph::element::i8}},
-                {1, {ngraph::element::i8}}
-            }),
-        });
+        std::vector<PrecisionsRestriction> supportedPrecisions{};
+        //Only enable conv/group conv signed input on AMX platform.
```

Comment:
Shouldn't we do it in scope of separate PR and benchmark it separately?
As far as I understand this PR is mostly about amx issues.
But here we are changing non-amx behavior as well (disabling s8 support)

Reply:
Yes. Since we have regression on VNNI when switching back to int8 input type. Switching back to U8 on VNNI needs to handle zero point and test. So that is why I add this part into this. Will update this PR description. In fact, we can use this PR to trigger related test for easiness. Thx!

Reply:
I mean yes, I understand the reason, but in general, I think it is much more convenient just to have a separate commit for this which we can refer to in case of issues.
Just to not mix all the fixes in scope of one commit / PR
@maxnick @dmitry-gorokhov What do you think?

Reply:
discussed, it seems we have no choice and have to do it in scope of single PR
So the comment can be ignored